pkg/tarmak/assets/assets.go

```diff
@@ -5,4 +5,4 @@
 
 package assets
 
-//go:generate go-bindata -prefix ../../../ -pkg $GOPACKAGE -o assets_bindata.go ../../../terraform/amazon/modules/... ../../../terraform/amazon/templates/... ../../../puppet/... ../../../packer/...
+//go:generate go-bindata -prefix ../../../ -pkg $GOPACKAGE -o assets_bindata.go ../../../wing_linux_amd64 ../../../terraform/amazon/modules/... ../../../terraform/amazon/templates/... ../../../puppet/... ../../../packer/...
```

Comment:
It would be great if we could have smaller release binaries that do not contain the wing_linux_amd64 built into.

I have had some branch that did not merge, which vendored other binaries using a devmode build tag:

https://github.com/jetstack/tarmak/pull/126/commits/d89de015fef66a33d100f0895801e031a20be72c

I should have remembered that before you started implenting it :grimacing: